### ISSUE TITLE
Github logging not working

### ISSUE THREAD

**Issue by rassidm** [callicoder/spring-boot-react-oauth2-social-login-demo#45]
Github logging not working
error msg :

Email not found from OAuth2 provider

**Comment by DarshanMaradiya** [comment]
Yeah Facing the same problem. I found the solution on [this](https://stackoverflow.com/questions/35373995/github-user-email-is-null-despite-useremail-scope), but don't know where and how to make extra call to get the email.

**Comment by gcc-2020-test** [comment]
I'm facing the same issue. Gives "Email not found from OAuth2 provider" error for a new Github app as well the one whose client id is mentioned in the code.

**Comment by akanksha3008** [comment]
I'm facing the same issue. Gives "Email not found from OAuth2 provider" error for both, new Github app as well as for the one whose client id is mentioned in the code.

**Comment by TCT2001** [comment]
Same Issue. Solve: Do not use condition StringUtils.isEmpty(oAuth2UserInfo.getEmail()) when checking github provider
